patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -1359,12 +1359,21 @@
 +        return false;
 +    }
 +
++    /**
++     * Set the PathNodeType for this block. null for vanilla behavior.
++     * @return the PathNodeType
++     */
++    @Nullable public net.minecraft.pathfinding.PathNodeType getAiPathNodeType()
```

Comment:
1. Annotation should be on a new line;
2. At least this should be state-aware; consider exposing the block positon.